ui/helpers/utils/notification.util.ts

```diff
@@ -420,9 +420,11 @@ export const getNetworkFees = async (notification: OnChainRawNotification) => {
   const rpcUrl = getRpcUrlByChainId(`0x${chainId}` as HexChainId);
   const connection = {
     url: rpcUrl,
-    headers: {
-      'Infura-Source': 'metamask/metamask',
-    },
+    headers: process.env.STORYBOOK_ENV
+      ? undefined
+      : {
+          'Infura-Source': 'metamask/metamask',
```

Comment:
This sub-optimal solution was previously introduced to handle a dedicated header for RPC calls to the Infura endpoint. In this PR, the use of this call is managed without the header in the case of a Storybook build

Reply:
Storybook registers certain global variables that you can check for during runtime. For instance, Storybook adds a window.__STORYBOOK_ADDONS_CHANNEL__ property when it runs.

You can use this to conditionally detect Storybook like so:
`const isStorybook = typeof window !== 'undefined' && !!window.__STORYBOOK_ADDONS_CHANNEL__;`

This way, you won’t need the STORYBOOK_ENV variable; instead, you’re checking for a runtime indication that you’re in Storybook.

Reply:
thx @andreahaku . I verified that Storybook indeed sets its own env, so I can check `process.env.STORYBOOK` without having to introduce a new ENV